src/ansys/aedt/core/hfss.py

```diff
@@ -3894,6 +3936,7 @@ def edit_source(self, assignment=None, power="1W", phase="0deg"):
         True
 
         """
+        warnings.warn("`edit_sources` is deprecated. Use `edit_sources` method instead.", DeprecationWarning)
 
```

Comment:
Since it is deprecated, shouldn't this method be implemented using `edit_sources` instead ? Like you did for `edit_sources_from_file`

Reply:
Done, the problem of this method is that it was not covering all the combination of edit sources